docs/user/index.md

```diff
@@ -242,8 +239,8 @@ def notifyAtomist(buildStatus, buildPhase="FINALIZED",
 }
 ```
 
-Then call `notifyAtomist` when the build starts, e.g., in the first
-stage, and ends, i.e., in the `post` block, sending the appropriate
+Then call `notifyAtomist` when the build starts (here, in the first
+stage) and ends ( in the `post` block), sending the appropriate
```

Comment:
"here" isn't right. If you are going to get rid of "e.g.", I'd just remove it.

There appears to be a space between the open parenthesis and "in the `post` block".

New text:  "... when the build starts (in the first stage) and ends (in the `post` block), ..."